src/garage/torch/policies/policy.py

```diff
@@ -20,18 +22,10 @@ def __init__(self, env_spec, name):
 
     @abc.abstractmethod
     def get_action(self, observation):
-        """Get a single action given an observation.
+        """Get action sampled from the policy.
 
         Args:
-            observation (torch.Tensor): Observation from the environment.
-
-        Returns:
-            tuple:
-                * torch.Tensor: Predicted action.
-                * dict:
-                    * list[float]: Mean of the distribution
-                    * list[float]: Log of standard deviation of the
-                        distribution
+            observation (np.ndarray): Observation from the environment.
```

Comment:
There probably still should be documentation of the return types (even if the current docstring is wrong, it should be `Tuple[np.ndarray, dict[str,np.ndarray]]`).